### ISSUE TITLE
Issue while sending JSON from react native to web by webview-bridge

### ISSUE THREAD

**Issue by tazmattaz** [alinz/react-native-webview-bridge#19]
Hello,

I am using your plugin from start. Previously it was working very fine but when I updated your latest version then I am facing below issues.

Issue1: If we send some simple json from react native to weView then it is working fine. But when I tried below JOSN (some dummy JSON) then it is not sending message to webView.

```js
var data = {  
            content:'<p>Story: <a href="https://www.google.com">https://www.google.com</a>\n</p>\n<p>1.For - Close (i.e by \'X\') does not change to hyper-link - Delivered.\n</p>'
		}
```

Issue 2: I sent message as object but when I got this from web then it is showing as string. 

Please can you check above issues?

Thanks in advance.

**Comment by alinz** [comment]
So you are sending `data` and the content that you are receiving is in string?
Can't you apply `JSON.parse` on the received data to convert to JSON object?

**Comment by tazmattaz** [comment]
Can't you apply JSON.parse on the received data to convert to JSON object?
Yes I am using this function to convert to JSON object. I have fixed fixed both issues at my end(with some patches). But this should not convert to string.

I have fixed previous issue as well but with patch.
First of all I converted my object into string and then call JavaScript function `escape(jsonstring)` and send to webView and in webView again I am doing revert on webView, In this way this is working.
But with previous version I was not using any kind of patch, I was sending JSON and getting JSON as well at webView side
